src/utils/deploy/upload-files.js

```diff
@@ -79,10 +85,12 @@ const retryUpload = (uploadFn, maxRetry) =>
 
     const tryUpload = async () => {
       try {
-        const results = await uploadFn()
+        const results = await uploadFn(retryCount)
         return resolve(results)
```

Comment:
[dust] new line before return

Reply:
Tasting my own poison! 🙈 

Done in 400da760aa283649d2145643f0cfcb11a0b4876c.